gcovr/formats/html/write.py

```diff
@@ -125,7 +135,7 @@ def load_css_template(options):
             template_path = os.path.relpath(options.html_css)
             return user_templates().get_template(template_path)
 
-        return templates().get_template("style.css")
+        return templates(options).get_template("style.css")
```

Comment:
I think with this we can remove the option `--html-css` because the user can provide one template directory where a `style.css` is present two have the same functionality.

Reply:
I definitely see your point, but wouldn't that confuse existing users who might only be doing CSS adjustments? 

To be clear, I'm happy to do the removal, just with it being such a small footprint of a feature I don't think the overlap of functionality is that bad.

Reply:
Since the new option will imply a fixed name I think it's better to keep the current behavior.